### DISCUSSION TITLE
Meow Mode Mascot

### DISCUSSION THREAD

**Discussion by TheTimeTombs** [meow-edit/meow#657] [Show and tell]
First, I would like to thank you for all of the hard work in creating this project. I stubbornly stuck to default Emacs keybindings for years but now I have seen the light. Running Emacs in a docker container meant I had to find a different way to access commands and I never gave Meow Mode a real attempt. Now that I have been using it, I see why it is so great. 

I was thinking. I love the icon on the home page since it has hints of the Emacs logo and also has the "M" shape. But, I thought that with how sublime Meow Mode is, that maybe it needs a mascot that represents the power of Meow Mode. Much like the Gnu for Emacs. I am not a artist, so here is an AI generated cat "deity" with a keyboard.

![image](https://github.com/user-attachments/assets/aacad909-cd6f-41ec-9279-4f40b1799305)

**Comment by DogLooksGood** [comment]
I like the concept. Hope someone who knows art can help.

![image](https://github.com/user-attachments/assets/7478c401-b6a1-4a10-b3a3-75d27ddf3568)

I'm too bad at drawing.

**Comment by gs-101** [comment]
It might seem a bit silly on my part, but looking at the current logo the first thing I think of are these types of shadow creatures. Something like this:

![image](https://github.com/user-attachments/assets/07189b03-b718-4bd4-aa0a-e7d46171913a)
https://www.artstation.com/artwork/Z2NqR

Maybe the logo could be used for the mouth or the eyes if following this type of design (though I think that the keyboard is an important piece lacking in this example image).

**Reply by gs-101** [reply]
![meow](https://github.com/user-attachments/assets/9626bbb5-885c-47a7-9e56-49807b78d2cd)

Example of a face.

**Reply by DogLooksGood** [reply]
LOL, I was using that M to represent the ear of cat.

**Reply by gs-101** [reply]
> LOL, I was using that M to represent the ear of cat.

Yeah I just noticed AFTER creating the image XD.

**Reply by DogLooksGood** [reply]
I'm thinking a similar M for the head, and use `EOW` in a squared font to represent the keyboard. Maybe add two eyes to make it looks more like a cat.  But this is an idea of logo, not a mascot.
